src/Mod/PartDesign/Gui/TaskPipeParameters.h

```diff
@@ -27,6 +27,7 @@
 #include <Gui/TaskView/TaskView.h>
 #include <Gui/Selection.h>
 #include <Gui/TaskView/TaskDialog.h>
+#include <Gui/Widgets.h>

```

Comment:
Further down the Gui::ButtonGroup is used as a pointer so that in the header file Gui/Widgets.h doesn't need to be included and a forward declaration suffices. This way the compilation time won't be negatively affected.